frontend/common/utils/format.js

```diff
@@ -112,6 +112,17 @@ const Format = {
     return value
   },
 
+  formatNumber(number) {
```

Comment:
What does this do? Can we add a comment or something?

Reply:
The function was renamed, and I added a comment. Also, I changed the behavior a bit.